README.rst

```diff
@@ -0,0 +1,591 @@
+tractor
+=======
+A minimalist `actor model`_ built on multiprocessing_ and trio_.
+
+|travis|
+
+.. |travis| image:: https://img.shields.io/travis/tgoodlet/tractor/master.svg
+    :target: https://travis-ci.org/tgoodlet/tractor
+
+``tractor`` is an attempt to take trionic_ concurrency concepts and apply
+them to distributed-multicore Python.
+
+``tractor`` lets you run and spawn Python *actors*: separate processes which are internally
+running a ``trio`` scheduler and task tree (also known as an `async sandwich`_).
+
+Actors communicate with each other by sending *messages* over channels_, but the details of this
+in ``tractor`` is by default hidden and *actors* can instead easily invoke remote asynchronous
+functions using *portals*.
+
+``tractor``'s tenets non-comprehensively include:
+
+- no spawning of processes *willy-nilly*; causality_ is paramount!
+- `shared nothing architecture`_
+- remote errors `always propagate`_ back to the caller
+- verbatim support for ``trio``'s cancellation_ system
+- no use of *proxy* objects to wrap RPC calls
+- an immersive debugging experience
+- be simple, be small
+
+.. warning:: ``tractor`` is in alpha-alpha and is expected to change rapidly!
+    Expect nothing to be set in stone and your ideas about where it should go
+    to be greatly appreciated!
+
+.. _actor model: https://en.wikipedia.org/wiki/Actor_model
+.. _trio: https://github.com/python-trio/trio
+.. _multiprocessing: https://docs.python.org/3/library/multiprocessing.html
+.. _trionic: https://trio.readthedocs.io/en/latest/design.html#high-level-design-principles
+.. _async sandwich: https://trio.readthedocs.io/en/latest/tutorial.html#async-sandwich
+.. _always propagate: https://trio.readthedocs.io/en/latest/design.html#exceptions-always-propagate
+.. _causality: https://vorpus.org/blog/some-thoughts-on-asynchronous-api-design-in-a-post-asyncawait-world/#c-c-c-c-causality-breaker
+.. _shared nothing architecture: https://en.wikipedia.org/wiki/Shared-nothing_architecture
+.. _cancellation: https://trio.readthedocs.io/en/latest/reference-core.html#cancellation-and-timeouts
+.. _channels: https://en.wikipedia.org/wiki/Channel_(programming)
+
+
+Install
+-------
+No PyPi release yet!
+
+::
+
+    pip install git+git://github.com/tgoodlet/tractor.git
+
+
+What's this? Spawning event loops in subprocesses?
+--------------------------------------------------
+Close, but not quite.
+
+The first step to grok ``tractor`` is to get the basics of ``trio``
+down. A great place to start is the `trio docs`_ and this `blog post`_
+by njsmith_.
+
+``tractor`` takes much inspiration from pulsar_ and execnet_ but attempts to be much more
+minimal, focus on sophistication of the lower level distributed architecture,
+and of course does **not** use ``asyncio``, hence **no** event loops.
```

Comment:
Rewrite this.

- This is a word salad (did you forget something): "... but attempts to be much more minimal, focus on sophistication of the ..."
- I'd suggest using "focus" instead of "minimal" - as minimal is subjective and "sophistication of the lower level distributed architecture" does not invoke ideas of minimalism - the opposite, in fact (not that there's anything wrong with **not** being minimal)
- Different idea, use a different sentence: "and of course does **not** use ``asyncio``, hence **no** event loops"